particle/pythia/pythiaid.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2018-2019, Eduardo Rodrigues and Henry Schreiner.
+#
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/particle for details.
+
+"""
+Class representing a Pythia ID.
+"""
+
+from __future__ import absolute_import
+
+import csv
+
+from .. import data
+from ..pdgid import PDGID
+from ..exceptions import MatchingIDNotFound
+
+
+with data.open_text(data, 'pdgid_to_pythiaid.csv') as _f:
+    _bimap = {int(v['PYTHIAID']):int(v['PDGID']) for v in csv.DictReader(_f)}
```

Comment:
Shouldn't this use a converter?

Reply:
I did this here because of (1) speed, because the bimaps call the 2 ID class constructors for each item, and (2) tp prevent a circular dependency, since the BiMap class imports also the 2 ID classes.